object-manipulation-3/main.js

```diff
@@ -0,0 +1,334 @@
+
+// Create an array of player objects that will represent each player
+// add a points property that we will be able to compare later
+var players = [
+  {
+    name: 'Brian',
+    hand: [],
+    points: 0
+  },
+  {
+    name: 'Ron',
+    hand: [],
+    points: 0
+  },
+  {
+    name: 'Brick',
+    hand: [],
+    points: 0
+  },
+  {
+    name: 'Veronica',
+    hand: [],
+    points: 0
+  }
+];
+
+// create a list of card objects
+var cards = [
+  {
+    rank: 'Ace',
+    suit: 'hearts'
+  },
+  {
+    rank: '2',
+    suit: 'hearts'
+  },
+  {
+    rank: '3',
+    suit: 'hearts'
+  },
+  {
+    rank: '4',
+    suit: 'hearts'
+  },
+  {
+    rank: '5',
+    suit: 'hearts'
+  },
+  {
+    rank: '6',
+    suit: 'hearts'
+  },
+  {
+    rank: '7',
+    suit: 'hearts'
+  },
+  {
+    rank: '8',
+    suit: 'hearts'
+  },
+  {
+    rank: '9',
+    suit: 'hearts'
+  },
+  {
+    rank: '10',
+    suit: 'hearts'
+  },
+  {
+    rank: 'Jack',
+    suit: 'hearts'
+  },
+  {
+    rank: 'Queen',
+    suit: 'hearts'
+  },
+  {
+    rank: 'King',
+    suit: 'hearts'
+  },
+  {
+    rank: 'Ace',
+    suit: 'spades'
+  },
+  {
+    rank: '2',
+    suit: 'spades'
+  },
+  {
+    rank: '3',
+    suit: 'spades'
+  },
+  {
+    rank: '4',
+    suit: 'spades'
+  },
+  {
+    rank: '5',
+    suit: 'spades'
+  },
+  {
+    rank: '6',
+    suit: 'spades'
+  },
+  {
+    rank: '7',
+    suit: 'spades'
+  },
+  {
+    rank: '8',
+    suit: 'spades'
+  },
+  {
+    rank: '9',
+    suit: 'spades'
+  },
+  {
+    rank: '10',
+    suit: 'spades'
+  },
+  {
+    rank: 'Jack',
+    suit: 'spades'
+  },
+  {
+    rank: 'Queen',
+    suit: 'spades'
+  },
+  {
+    rank: 'King',
+    suit: 'spades'
+  },
+  {
+    rank: 'Ace',
+    suit: 'diamonds'
+  },
+  {
+    rank: '2',
+    suit: 'diamonds'
+  },
+  {
+    rank: '3',
+    suit: 'diamonds'
+  },
+  {
+    rank: '4',
+    suit: 'diamonds'
+  },
+  {
+    rank: '5',
+    suit: 'diamonds'
+  },
+  {
+    rank: '6',
+    suit: 'diamonds'
+  },
+  {
+    rank: '7',
+    suit: 'diamonds'
+  },
+  {
+    rank: '8',
+    suit: 'diamonds'
+  },
+  {
+    rank: '9',
+    suit: 'diamonds'
+  },
+  {
+    rank: '10',
+    suit: 'diamonds'
+  },
+  {
+    rank: 'Jack',
+    suit: 'diamonds'
+  },
+  {
+    rank: 'Queen',
+    suit: 'diamonds'
+  },
+  {
+    rank: 'King',
+    suit: 'diamonds'
+  },
+  {
+    rank: 'Ace',
+    suit: 'clubs'
+  },
+  {
+    rank: '2',
+    suit: 'clubs'
+  },
+  {
+    rank: '3',
+    suit: 'clubs'
+  },
+  {
+    rank: '4',
+    suit: 'clubs'
+  },
+  {
+    rank: '5',
+    suit: 'clubs'
+  },
+  {
+    rank: '6',
+    suit: 'clubs'
+  },
+  {
+    rank: '7',
+    suit: 'clubs'
+  },
+  {
+    rank: '8',
+    suit: 'clubs'
+  },
+  {
+    rank: '9',
+    suit: 'clubs'
+  },
+  {
+    rank: '10',
+    suit: 'clubs'
+  },
+  {
+    rank: 'Jack',
+    suit: 'clubs'
+  },
+  {
+    rank: 'Queen',
+    suit: 'clubs'
+  },
+  {
+    rank: 'King',
+    suit: 'clubs'
+  }
+];
```

Comment:
Rather than writing out the entire deck, I would create an array of all possible card suits and another array of all possible card values, then use those to generate a deck of cards.